xbmc/pvr/timers/PVRTimerInfoTag.cpp

```diff
@@ -879,6 +879,18 @@ bool CPVRTimerInfoTag::HasEpgInfoTag(void) const
   return m_epgTag != NULL;
 }
 
+bool CPVRTimerInfoTag::HasSeriesEpgInfoTag(void) const
+{
+  if (m_epgTag &&
```

Comment:
don't take this comment as a review btw, just happened to see this one when clicking on the wrong link, still no time to review stuff ;-)